### PR TITLE
Landing page wip

### PR DESCRIPTION
**Pull request by reedwane**

### CODE REVIEW

**Comment by TopRateANZ** [comment]
There is no Screenshort on this PR

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+WORKDIR /app
+
+# Install dependencies based on the preferred package manager
+COPY package.json yarn.lock* package-lock.json* pnpm-lock.yaml* ./
+RUN \
+  if [ -f yarn.lock ]; then yarn --frozen-lockfile; \
+  elif [ -f package-lock.json ]; then npm ci; \
+  elif [ -f pnpm-lock.yaml ]; then yarn global add pnpm && pnpm i --frozen-lockfile; \
```
Comment (TopRateANZ) [inline Dockerfile:14]: Clean Up

**Comment by TopRateANZ** [comment]
Opacity is overshadowing content . The hero text should be on 2 lines not 3 and the size should be reduced. You can look at 
https://topratetransfer.com/#/  as a Guide

**Comment by TopRateANZ** [comment]
Size or Zoom Issues on Landing Page

**Comment by TopRateANZ** [comment]
FORGET PASSWORD Page Should have a Country Dropdown on the Left where country code is selected . If it's possible for the dropdown

**Comment by TopRateANZ** [comment]
Clicking on the Logo Should Redirect to the Landing Page

**Comment by TopRateANZ** [comment]
Drop dpown for Sending Country Should look like https://wise.com/  atleast 60-70% of the width

**Comment by TopRateANZ** [comment]
All Dropdowns should have padding on the right

**Comment by reedwane** [comment]
image

I took care of the opacity as requested, and adjusted the currency dropdown

**Comment by reedwane** [comment]
image

I have adjusted the hero as requested. Also adjusted the opacity of the overlay on the video.

**Pull request merged**